### ISSUE TITLE
Should the extension just look at the global `angular`?

### ISSUE THREAD

**Issue by mgol** [IgorMinar/ng-detector#1]
I have an app in Angular 1.3-rc.5 (not public yet) that has debug info turned off and uses manual bootstrap. That causes this extension to not recognize it; this also might be a more common issue once the stable 1.3 is out.

Wouldn't it catch more scenarios and be simpler to implement to just look for the global named `angular`, perhaps also checking for existence of some of its methods?

Alternatively, if all current measures fail and there is a global named `angular` with a method called `bootstrap` it could be monkey-patched to listen for the invocation of the method and register it as an Angular app then?

**Comment by IgorMinar** [comment]
Yes, but accessing global window from an extension is not straightforward as far as I can tell. Brian promised to send a PR to do this using a trick that Batarang uses.

**Comment by IgorMinar** [comment]
@btford

**Comment by btford** [comment]
On it.

**Comment by IgorMinar** [comment]
So Brian merged his thing in already, but currently the code looks for angular in global context only if we have an indication that there is angular on the page (via script url or angular specific attributes).

Is this good enough?

I'd prefer to avoid injecting scripts to all the pages because that's intrusive and might slow down pages that don't use angular. Could we just improve the lightweight heuristics to catch your scenario?
